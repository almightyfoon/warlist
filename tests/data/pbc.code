c6501b_-0J5p8i8j5Z5Z5Z5Z5Z7j3WfC7y9uirirchch8M___-089p5Z5Z5Z8P7m7y7AiBiBgFgGgHiKiL
