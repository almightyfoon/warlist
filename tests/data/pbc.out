https://conflictchamber.com/#c6501b_-0J5p8i8j5Z5Z5Z5Z5Z7j3WfC7y9uirirchch8M___-089p5Z5Z5Z8P7m7y7AiBiBgFgGgHiKiL

Mercenary Army - 75 / 75 points
[Theme] The Irregulars

[Magnus 2] Magnus the Warlord [+28]
 - Buccaneer [6]
 - Freebooter [9]
 - Nomad [11]
 - Nomad [11]
 - Nomad [11]
 - Nomad [11]
 - Nomad [11]
Anastasia di Bray [3]
Eiryss, Mage Hunter of Ios [0(7)]
Hutchuk, Ogrun Bounty Hunter [0(6)]
Orin Midwinter, Rogue Inquisitor [0(5)]
Ragman [0(4)]
Horgenhold Artillery Corps [6]
Horgenhold Artillery Corps [6]
Kayazy Eliminators [5]
Kayazy Eliminators [5]
Lady Aiyana & Master Holt [8]

Mercenary Army - 75 / 75 points
[Theme] The Kingmaker's Army

[Damiano 1] Captain Damiano [+28]
 - Nomad [11]
 - Nomad [11]
 - Nomad [11]
Dirty Meg [3]
Kell Bailoch [0(5)]
Orin Midwinter, Rogue Inquisitor [0(5)]
Stannis Brocker [0(7)]
Steelhead Halberdiers (max) [11]
Steelhead Halberdiers (max) [11]
Trencher Infantry (max) [16]
 - Trencher Infantry Officer & Sniper [5]
 - Trencher Infantry Rifle Grenadier (1) [2]
Trencher Long Gunners (max) [18]
 - Trencher Long Gunner Officer [4]
